app/src/main/java/fr/free/nrw/commons/contributions/MainActivity.java

```diff
@@ -238,7 +238,15 @@ private void initMain() {
     public void onBackPressed() {
         if (contributionsFragment != null && activeFragment == ActiveFragment.CONTRIBUTIONS) {
             // Meas that contribution fragment is visible
-            contributionsFragment.backButtonClicked();
+            if (ContributionsFragment.mediaDetailPagerFragment==null) { //means you open the app currently and not open mediaDetailPage fragment
```

Comment:
This is not a good practice to make a fragment static and reaching it with classname. You should use object oriented approaches instead. Otherwise it will cause leaks.

Reply:
wait wait, sorry this seems like resolved in current version.